internal/platform/utimes_unsupported.go

```diff
@@ -0,0 +1,12 @@
+//go:build !windows
+
+package platform
+
+import "syscall"
+
+func futimens(fd uintptr, atimeNsec, mtimeNsec int64) error {
+	// Go exports syscall.Futimes, which is microsecond granularity, and
```

Comment:
@evacchi are you interested in trying to implement calling futimens on darwin, freebsd and linux? It would be cleaner for sure.

Reply:
sure why not 💪

Reply:
#1205 will resolve this!